### ISSUE TITLE
Create redirects for 2023 talks; Add solid URL concept for 2024

### ISSUE THREAD

**Issue by tordans** [openstreetmap-polska/sotm-eu-2024#11]
Hey! I was sad to see that "old" URLs from 2023 don't work anymore.

A golden rule is, "cool URLs don't change" https://www.w3.org/Provider/Style/URI

Example: https://stateofthemap.eu/program/ungap-the-map-designing-moonshot-cycling-networks

Where do I find the content of the 2023 SOTM?
And what is needed to add redirects for the old URLs to the new place of the old page?
And also, what is your concept for 2024 to not run into the same issue? I don't see any date-part in the URL ATM, so it looks like we will have the same issue next year?

**Comment by joostschouppe** [comment]
Indeed, we should have put the 2023 website in 2023.sotm from the start... The pages are still available via e.g. https://2023.stateofthemap.eu/program/ungap-the-map-designing-moonshot-cycling-networks . Ping @jbelien , do you have an idea for a fix?

**Comment by tordans** [comment]
> The pages are still available via e.g. https://2023.stateofthemap.eu/program/ungap-the-map-designing-moonshot-cycling-networks .

Thanks, that is good to know.

I think it would be ideal to add a static list of redirects from the old URLs to that new URL. Those URLs would be blocked from being used for the new website, but first that content should be scoped to 2024 from the start and second the URLs are mostly very specific. Special cases would be /program and such.

**Comment by jbelien** [comment]
OSM Poland has now the hand on stateofthemap.eu.
They can add in the 2024 version of the website a redirect to 2023 for "our" URLs but that will be quite some work I think.

**Comment by ttomasz** [comment]
Creating redirects for all of the previous urls seems like a lot of work for little gain. It's probably best to pass on advice to future organizers of 2025 edition so future editions can handle it better and accept that 2023 and 2024 require manual url changes to see historical sites.
